azure_sdk/azure_sdk.gemspec

```diff
@@ -32,40 +32,55 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'azure_mgmt_container_registry', "~>#{version}"
+  spec.add_runtime_dependency 'azure_mgmt_customer_insights', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_datalake_analytics', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_datalake_store', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_devtestlabs', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_dns', "~>#{version}"
+  spec.add_runtime_dependency 'azure_mgmt_event_grid', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_event_hub', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_features', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_graph', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_monitor', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_iot_hub', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_key_vault', "~>#{version}"
+  spec.add_runtime_dependency 'azure_mgmt_links', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_locks', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_logic', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_machine_learning', "~>#{version}"
+  spec.add_runtime_dependency 'azure_mgmt_managed_applications', "~>#{version}"
+  spec.add_runtime_dependency 'azure_mgmt_marketplace_ordering', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_media_services', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_mobile_engagement', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_network', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_notification_hubs', "~>#{version}"
+  spec.add_runtime_dependency 'azure_mgmt_operational_insights', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_policy', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_powerbi_embedded', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_recovery_services', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_recovery_services_backup', "~>#{version}"
+  spec.add_runtime_dependency 'azure_mgmt_recovery_services_site_recovery', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_redis', "~>#{version}"
+  spec.add_runtime_dependency 'azure_mgmt_relay', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_resources', "~>#{version}"
+  spec.add_runtime_dependency 'azure_mgmt_resources_management', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_scheduler', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_search', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_server_management', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_service_bus', "~>#{version}"
+  spec.add_runtime_dependency 'azure_mgmt_service_fabric', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_sql', "~>#{version}"
+  spec.add_runtime_dependency 'azure_mgmt_stor_simple8000series', "~>#{version}"
```

Comment:
`azure-storage` version [0.14.0.preview](https://rubygems.org/gems/azure-storage/versions/0.14.0.preview) has been released. We should update to that as usual. :)

Reply:
Updating this, thanks!